src/tests/test_jsproxy.py

```diff
@@ -361,6 +362,23 @@ def test_call_pyproxy_destroy_args(selenium):
         """
     )
 
+    selenium.run_js(
+        r"""
+        let y;
+        pyodide.setDebug(false);
+        self.f = function(x){ y = x; }
+        pyodide.runPython(`
+            from js import f
+            f({})
+            f([])
+        `);
+        assertThrows(() => y.length, "Error",
+            "This borrowed proxy was automatically destroyed at the end of a function call.*\n" +
+            'For more information about the cause of this error, use `pyodide.setDebug.true.`'
```

Comment:
```suggestion
            'For more information about the cause of this error, use `pyodide.setDebug(true)`'
```

Reply:
The regex won't match then since it makes a regex group. I think you need `\\(`. But it's annoying to debug these regex...